src/metax_api/api/rest/base/views/file_view.py

```diff
@@ -173,9 +173,13 @@ def datasets(self, request):
         parameters in GET. Also, some clients forcibly shove parameters in body in GET
         requests to query parameters, so using POST instead is more guaranteed to work.
         """
+        keysonly = CommonService.get_boolean_query_param(self.request, 'keysonly')
 
-        if CommonService.get_boolean_query_param(request, 'detailed'):
```

Comment:
?detailed is taken away because it does the same thing as the new ?keys=files, if I understood this correctly https://jira.eduuni.fi/browse/CSCFAIRMETA-636

Reply:
True, but we should keep the existing implementation working until we are sure that other services have adopted the new parameters

Reply:
Äh you're right, will restore ?detailed parameter